testbed/testbed/test_case.go

```diff
@@ -121,7 +122,7 @@ func NewTestCase(
 	require.NoErrorf(t, os.MkdirAll(tc.resultDir, os.ModePerm), "Cannot create directory %s", tc.resultDir)
 
 	// Set default resource check period.
-	tc.resourceSpec.ResourceCheckPeriod = 3 * time.Second
+	tc.resourceSpec.ResourceCheckPeriod = 1 * time.Second
```

Comment:
This an cause problems during startup when we have a spike of CPU consumption. Why is this change needed?

Reply:
Reverted, was looking for error cause.